src/rgw/support/rgw_support_bundle

```diff
@@ -157,6 +157,9 @@ class RGWSupportBundle:
             help="Include/Exclude stacktrace, Default = False")
         parser.add_argument('--modules', dest='modules',
             help="list of components & services to generate support bundle.")
+        parser.add_argument('--all', type=RGWSupportBundle.str2bool, default=False,
+            help="Include/Exclude all logs[stacktrace, Coredumps, Binary], Default = False")
```

Comment:
help="Include all debug data, including logs, config, stack traces, core dumps, binaries, etc, possibly resulting in much HEAVIER support bundle], Default = False")

Reply:
@rohit-k-dwivedi add ticket id to commit msg

Reply:
done